environment.yml

```diff
@@ -0,0 +1,10 @@
+name: IvadoMedEnv
+channels:
+  - conda-forge
+  - pytorch
+dependencies:
+  - python>=3.6, < 3.9
+  - anaconda
```

Comment:
@dyt811 can you explain why you depend on `anaconda` explicitly?

This doesn't seem to be compatible with https://github.com/conda-forge/miniforge, which is what I picked to use in https://github.com/neuropoly/computers/pull/120/ on the theory that it would be smaller and better supported.

Removing this one line seems to make ivadomed fully compatible with miniforge.

Reply:
Yup. No need to use anaconda. Miniconda is good enough. 👍